app/src/main/res/layout/search_toolbar.xml

```diff
@@ -20,7 +34,19 @@
             android:layout_toEndOf="@+id/ivBack"
             android:ems="10"
             android:hint="search"
-            android:inputType="textPersonName" />
+            android:inputType="textPersonName"/>
+
+<!--        <AutoCompleteTextView-->
```

Comment:
If you've committed this code before, feel free to delete this because GitHub should maintain an old revision for you to reference if you ever need it